src/components/ArticleId.jsx

```diff
@@ -26,14 +27,26 @@ function ArticleId() {
         setIsCommentsLoading(true);
         fetchComments(article_id)
           .then((comments) => {
-            console.log(comments);
-            setComments([...comments]);
+              const formattedComments = comments.map(comment => {
+                  const formattedDate = new Date(comment.created_at)
+                  return {...comment, formatted_date: formattedDate.toLocaleString("en-GB")}
+              })
+
+            setComments([...formattedComments]);
             setIsCommentsLoading(false);
           });
       }, [article_id]);
 
     const Comment = <section className="comments"> 
```

Comment:
Rather than putting this into a variable, this feels like a great moment to extract it into a new React Component, maybe called `CommentCard` or something similar?